src/Microsoft.AspNetCore.Rewrite/Internal/UrlActions/RedirectAction.cs

```diff
@@ -53,7 +53,14 @@ public override void ApplyAction(RewriteContext context, MatchResults ruleMatch,
                 pattern = Uri.EscapeDataString(pattern);
             }
 
-            if (pattern.IndexOf("://", StringComparison.Ordinal) == -1 && !pattern.StartsWith("/"))
+            if (string.IsNullOrEmpty(pattern))
+            {
+                response.Headers[HeaderNames.Location] = string.Empty;
```

Comment:
Also not valid. Needs to be a path.

As above, when would this condition trigger? Seems like this might be an error. I want to make sure inferring a redirect to '/' is correct